protos/index_old.proto

```diff
@@ -39,4 +39,6 @@ message InvertedIndexDetails {
   uint32 min_ngram_length = 9;
   uint32 max_ngram_length = 10;
   bool prefix_only = 11;
+  optional string tokenizer_plugin_library = 12;
+  optional string tokenizer_plugin_config = 13;
```

Comment:
This is sort of a change to the format, so we should probably open a design discussion on this. I can do that.

Reply:
Started a discussion here: https://github.com/lance-format/lance/discussions/5607